lib/devicetrust/enroll/enroll_test.go

```diff
@@ -37,25 +38,61 @@ func TestRunCeremony(t *testing.T) {
 
 	macOSDev1, err := testenv.NewFakeMacOSDevice()
 	require.NoError(t, err, "NewFakeMacOSDevice failed")
+	windowsDev1 := testenv.NewFakeWindowsDevice()
 
 	tests := []struct {
-		name string
-		dev  fakeDevice
+		name            string
+		dev             fakeDevice
+		assertErr       func(t *testing.T, err error)
+		assertGotDevice func(t *testing.T, device *devicepb.Device)
 	}{
 		{
-			name: "macOS device",
+			name: "macOS device succeeds",
 			dev:  macOSDev1,
+			assertErr: func(t *testing.T, err error) {
+				assert.NoError(t, err, "RunCeremony returned an error")
+			},
+			assertGotDevice: func(t *testing.T, d *devicepb.Device) {
+				assert.NotNil(t, d, "RunCeremony returned nil device")
+			},
+		},
+		{
+			name: "windows device succeeds",
+			dev:  windowsDev1,
+			assertErr: func(t *testing.T, err error) {
+				assert.NoError(t, err, "RunCeremony returned an error")
+			},
+			assertGotDevice: func(t *testing.T, d *devicepb.Device) {
+				require.NotNil(t, d, "RunCeremony returned nil device")
+				require.NotNil(t, d.Credential)
+				assert.Equal(t, windowsDev1.CredentialID, d.Credential.Id)
+			},
+		},
+		{
+			name: "linux device fails",
+			dev:  testenv.NewFakeLinuxDevice(),
+			assertErr: func(t *testing.T, err error) {
+				require.Error(t, err)
+				assert.True(
+					t, trace.IsBadParameter(err), "RunCeremony did not return an error of the correct type",
+				)
+				assert.ErrorContains(t, err, "linux")
+			},
+			assertGotDevice: func(t *testing.T, d *devicepb.Device) {
+				assert.Nil(t, d)
```

Comment:
```suggestion
				assert.Nil(t, d, "RunCeremony returned an unexpected, non-nil device")
```

Not asserting anything could be OK too.